### PR TITLE
Wordpress + mod_pagespeed

### PR DESCRIPTION
**Pull request by eduha**
The mod_pagespeed included to the app increases website loading speed on-the-fly. No extra configuration needed.

The one-click app also ships with YoastSEO and W3 Total Cache plugins. Redis server in installed and enabled for wordpress object cache in orders to increase performance.

### CODE REVIEW

File: cb-wordpress-pagespeed/stackscript.sh [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+# <UDF name="new_username" label="Linux User Name" example="webdev" />
+# <UDF name="new_password" label="Linux User Password" />
+# <UDF name="website" label="Website Domain" example="example.com" />
+# <UDF name="website_name" label="Wordpress Website Name" />
+# <UDF name="website_username" label="Wordpress Admin Username" />
+# <UDF name="website_password" label="Wordpress Admin Password" />
+# <UDF name="website_email" label="Wordpress Admin Email" />
+# <UDF name="website_locale" label="Wordpress Locale" example="Example: de_DE, nl_NL, ru_RU" default="en_US" />
+
```
Comment (hmorris3293) [inline cb-wordpress-pagespeed/stackscript.sh:13]: Add logging for future troubleshooting

`# Add Logging to /var/log/stackscript.log for future troubleshooting`
`exec > >(tee /dev/ttyS0 /var/log/stackscript.log) 2>&1`

**Review by hmorris3293 — COMMENTED** [review]
hi @eduha Thank you for the submission!  I've made a suggestion to the install script but overall the app works as expected! The last part of the submission here that is missing is the documentation piece for your Marketplace app. You can find an example of what you'll need to include in the link below:

https://github.com/linode/docs/blob/develop/docs/guides/platform/marketplace/harbor-marketplace-app/index.md

If you have any questions, please feel free to reach out

File: cb-wordpress-pagespeed/stackscript.sh (hunk 1 above)
Comment (eduha) [reply cb-wordpress-pagespeed/stackscript.sh:13]: Thanks @hmorris3293 !

I committed the change you suggested. Can you review it please?

**Comment by hmorris3293** [comment]
HI @eduha Sorry for the delay! Our team has mod_pagespeed on our list for add ons for our current Wordpress Marketplace app. We're currently working on doing some updates for that but we don't have an exact ETA. For now, we are going to deny this pull request, but we will look to add mod_pagespeed to our existing Wordpress app in the near future.
Thank you for the submission

**Pull request closed without merging**